vrk-kpa/suomidigi
# SUOPA-715 Frontpage

To test:
- `make fresh`
- Log in and go to https://suomidigi.docker.sh/node/7/edit
   - Select the "Hero" checkbox and add a hero paragraph to the page. 
   - Do not add a background image, but fill in all the other fields. You can copy the texts from here: https://xd.adobe.com/view/c1b40caa-b2e0-43a5-78d8-9690a43f540c-b785/screen/7c1dc8eb-0b46-4aa5-b434-be0fbcd8fc0c/Suomidigi-etusivu-launch
   - _You might need to add Liftup styles for all liftups on the page. Select either one. 2col or 3col._
   - Check that the layout is similar to proto, and that the mobile layout works as well. You can also try to add a background image for the hero.
- Edit the node again and modify the first Liftup reference (Nostot) paragraph. 
   - Add a title for it **Nyt käynnissä** and a link with title **Katso kaikki käynnissäolevat**. 
   - Save the node and check that the layout is similar to the proto. Also check that mobile width doesn't break the layout
- Edit the node again and add new paragraph View (näkymä).
   - Add title, description and link. You can copy them from the proto
   - Select view Groups (ryhmät) and view mode Groups | grid (ruudukko)
   - Save the node and check that the layout is similar to the proto. Also check that mobile width doesn't break the layout
- Lastly, go to https://suomidigi.docker.sh/node/add/event and create few events with minimum details
- Edit the front page node again and add new paragraph View (näkymä).
   - Add title, description and link. You can copy them from the proto
   - Select view Events and view mode Events | grid
   - Save the node and check that the layout is similar to the proto. Also check that mobile width doesn't break the layout
